app/models/follow.php

```diff
@@ -0,0 +1,43 @@
+<?php
+class Follow extends AppModel
+{
+    CONST TABLE = 'follow';
+    public static function unsetFollow($thread_id, $user_id)
+    {
+        $db = DB::conn();
+        $db->query("DELETE FROM follow where thread_id = ? AND user_id = ?", array($thread_id, $user_id));
```

Comment:
Done po :smile: I applied it to all po. https://github.com/renzosunico/MyClassroom/commit/e0697fe0a4d74285a2d84de23a870cdba107f333